whois-db/src/main/java/net/ripe/db/whois/db/WhoisServer.java

```diff
@@ -50,6 +54,15 @@ public void run() {
 
         whoisServer.start();
 
+        final MutablePropertySources sources = applicationContext.getEnvironment().getPropertySources();
+        StreamSupport.stream(sources.spliterator(), false)
+                .filter(ps -> ps instanceof EnumerablePropertySource)
+                .map(ps -> ((EnumerablePropertySource) ps).getPropertyNames())
+                .flatMap(Arrays::stream)
+                .distinct()
+                .filter(prop -> !(prop.contains("credentials") || prop.contains("password")))
```

Comment:
Can you log the property, but put in "***" or something for the value? As it is, nothing is logged, and so you can't tell whether the property is defined or not.

Reply:
done